pkg/model/project.go

```diff
@@ -410,13 +386,13 @@ func (p *Project) GetAllUserGroups() []*ProjectUserGroup {
 			Role:     builtinRBACRoleAdmin.String(),
 		})
 	}
-	if rbac.Editor != "" {
+	if rbac.Editor != "" && rbac.Editor != rbac.Admin {
```

Comment:
sorry, but what is the point of this condition? 🤔  I can not understand why we need this. If possible, add a comment for this would help a lot.

Reply:
True, I added a comment into this method.
https://github.com/pipe-cd/pipecd/pull/3581/commits/0f83fa3402a1a15ae7b2c9cba40b57d3205d97ea#diff-82a82b917b5274a349044117c5792800a8aa363e471febc0c0c3fed68f3c4b50R375